apps/wallet-dashboard/components/Dialogs/Staking/StakeDialog.tsx

```diff
@@ -166,19 +161,15 @@ export function StakeDialog({
             },
             {
                 onSuccess: (tx) => {
-                    if (onSuccess) {
-                        onSuccess(tx.digest);
-                    }
+                    addNotification('Stake transaction has been sent');
+                    setTxDigest(tx.digest);
+                    setView?.(StakeDialogView.TransactionDetails);
```

Comment:
does it make sense that `setView` is optional now?
I think the details view will always be displayed